### ISSUE TITLE
Specified Field tensor_type torch.float32 can not be used with use_vocab=False because we do not know how to numericalize it.

### ISSUE THREAD

**Issue by caolusg** [pytorch/text#433]
The output/label of the training set is a float type, I want to use it directly but got the issue. how to solve it?

**Comment by mttk** [comment]
That's odd, as `torch.float32` is in the `dtypes` of a `Field`. Can you paste the code snippet where you create the field?

**Comment by caolusg** [comment]
> That's odd, as `torch.float32` is in the `dtypes` of a `Field`. Can you paste the code snippet where you create the field?

hi, thank you for your reply, I have found the issue, I should put torch.FloatTensor but I put tensor.float
